Base/etc/fstab

```diff
@@ -1,10 +1,12 @@
 # Root file system. This is a fake entry which gets ignored by `mount -a`;
 # the actual logic for mounting root is in the kernel.
-/dev/hda	/	ext2	nodev,nosuid
-# Remount /bin and /dev while adding the appropriate permissions.
-/dev	/dev	bind	bind,nosuid
-/bin	/bin	bind	bind,nodev
+/dev/hda	/	ext2	nodev,nosuid,ro
+# Remount /bin, /dev, /root, and /home while adding the appropriate permissions.
```

Comment:
Shouldn't `/etc` be rw for root?
Similarly, I don't understand why we're limiting `/usr` to readonly

Or am I completely misunderstanding this?

Reply:
You are misunderstanding this.

Are you familiar with Android internals? Or SIP on Darwin/macOS? This is something similar.

If you do need to edit `/etc` or install new software into root, you can explicitly opt into modifying it:

```sh
$ chroot / -o rw
$ te /etc/foo
```

but by default, even processes running as root cannot modify those files.

Reply:
Oh, alternatively you could bind-mount `/etc` as writable when you need it:

```sh
$ mount /etc /etc -o bind
```

Reply:
Oh I see.
would be a bit weird as I don't think we're interested in _not_ letting the user change their configs easily, but I guess a mount for editing is a fine step to go through.